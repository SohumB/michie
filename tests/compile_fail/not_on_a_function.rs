use caching::caching;

#[caching(key_type = usize, key_expr = a)]
struct A;

fn main() {}
