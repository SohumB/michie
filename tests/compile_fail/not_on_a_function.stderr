error: expected `fn`
 --> tests/compile_fail/not_on_a_function.rs:4:1
  |
4 | struct A;
  | ^^^^^^
